pkg/process/checks/allprocesses_windows.go

```diff
@@ -382,53 +275,32 @@ type cachedProcess struct {
 	parsedArgs     []string
 }
 
-func (cp *cachedProcess) fill(proc *Win32_Process) (err error) {
-	// 0x1000 is PROCESS_QUERY_LIMITED_INFORMATION, but that constant isn't
-	// defined in x/sys/windows
-	cp.procHandle, err = windows.OpenProcess(0x1000, false, uint32(proc.ProcessID))
-	if err != nil {
-		log.Debugf("Couldn't open process %v %v", proc.ProcessID, err)
-		return err
-	}
-	var usererr error
-	cp.userName, usererr = getUsernameForProcess(cp.procHandle)
-	if usererr != nil {
-		log.Debugf("Couldn't get process username %v %v", proc.ProcessID, err)
-	}
-
-	cp.executablePath = *proc.ExecutablePath
-	if len(cp.executablePath) == 0 {
-		// some system processes don't give us the executable path variable.  Just
-		// give the executable name
-		cp.executablePath = proc.Name
-		log.Debugf("Setting alternate executable path (name) %d %s", proc.ProcessID, cp.executablePath)
-	}
-	cp.commandLine = *proc.CommandLine
-	var parsedargs []string
-	if len(cp.commandLine) == 0 {
-		parsedargs = append(parsedargs, cp.executablePath)
-	} else {
-		parsedargs = parseCmdLineArgs(cp.commandLine)
-	}
-	cp.parsedArgs = parsedargs
-	return
-}
-
 func (cp *cachedProcess) fillFromProcEntry(pe32 *w32.PROCESSENTRY32) (err error) {
 	// 0x1000 is PROCESS_QUERY_LIMITED_INFORMATION, but that constant isn't
+	// 0x10   is PROCESS_VM_READ
 	// defined in x/sys/windows
-	cp.procHandle, err = windows.OpenProcess(0x1000, false, uint32(pe32.Th32ProcessID))
+	cp.procHandle, err = windows.OpenProcess(0x1010, false, uint32(pe32.Th32ProcessID))
 	if err != nil {
-		log.Infof("Couldn't open process %v %v", pe32.Th32ProcessID, err)
-		return err
+		log.Debugf("Couldn't open process with PROCESS_VM_READ %v %v", pe32.Th32ProcessID, err)
+		cp.procHandle, err = windows.OpenProcess(0x1000, false, uint32(pe32.Th32ProcessID))
+		if err != nil {
+			log.Debugf("Couldn't open process %v %v", pe32.Th32ProcessID, err)
```

Comment:
Should we keep the status quo and do a log.Info, or was that too noisy?

Reply:
I think it would occur once per run (for PID 4, the system process).  I can put it back to info and try it if we want.

Reply:
Let's leave it out!